packages/site/src/layouts/base.astro

```diff
@@ -25,7 +26,7 @@ const navData = searchData.map(o => pick(o, ["name", "subCategories"]))
 		
 		<meta name="robots" content="noindex">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<SocialMetaTags title={title} siteName={import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK} />
+		<SocialMetaTags title={`${currentCategory} | ${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}.framework.dev`} siteName={import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK} />
```

Comment:
One more small change. The title is properly `titleCase`d when viewing the main content but is lowercase in the social card's title:

![image](https://user-images.githubusercontent.com/9042219/200383786-9587d3eb-dcab-44bb-9b7c-0700d42e6a83.png)